bin/test-lxd-storage-vm

```diff
@@ -357,6 +357,11 @@ do
         [ $(($(lxc exec v2 -- blockdev --getsize64 /dev/sda)/${GiB})) -eq "7" ]
         lxc delete -f v2
 
+        echo "==> Copy to different storage pool with same driver"
+        lxc storage create "${poolName}3" "${poolDriver}" size=20GiB
```

Comment:
Please can the storage pool be deleted after so the test can be run repeatedly locally. Thanks